_posts/2020-08-18-wtf-is-white-culture.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "WTF is White Culture"
+date: "2020-08-18 16:19:10 -0500"
+---
+
+At [work](http://underarmour.com) the other day, we had another session in a
+talk series the company has put together on anti-racism (and sometimes straying
+into other anti-oppression work). It's been really good stuff, which defies my
+expectations for how Corporate America™ generally handles these topics.
+
+This time, part of the session was Mary-Frances Winters talking about the idea
+of "cultural competency" and one of the things she said struck me. It was the
+idea that in order to gain strong cultural competency, it was important to
+become aware of your own culture and others' so that you could be mindful about
+the differences between them and how it might or should affect interactions with
+people.
+
+And it linked up with this idea that's been in my head for a while, which I
+don't remember where I originally heard it: That whiteness is not a culture.
+Which is to say, whiteness is only about power. It doesn't have folk music, or a
+traditional cuisine. There is no traditional ceremonial dress or religious
+practice. There is not culture of whiteness that isn't actually a culture of
+white supremacy.
+
+Since whiteness was forged out of a common European identity (which, go research
+some about why the idea of "Europe" is, itself, racist), it also means that the
+distinct cultural markers and practices necessarily had to be filed off to erase
+the differences. This might sound like "the melting pot" that the US often
+boasts of, and I think it is related. But I also think it is violence. Like many
+of the evils that whiteness has inflicted on those excluded from it, there is a
+mirror version that it has inflicted on those included.
+
+In particular within the US (which is the only place I've lived and really
+experienced whiteness, so the only place I'm really qualified to address), I
+think the twin projects of the eradication and oppression of the indigenous
+people, and the kidnapping and enslavement of Black people were important
+factors that are pretty specific to the US context. And I think the erasure of
+the cultures from which whiteness was made was key in white people justifying
+these two atrocities to themselves.
```

Comment:
And it’s important to note also that this continues to the present day through the constant appropriation and plundering that white people perform on other cultures. Recently I read [an interesting article from Creative Commons](https://creativecommons.org/2019/01/30/jane-anderson/) about how because (white) copyright views certain things produced by Indigenous culture as belonging to the “public domain”, they see it as belonging to the world (really, Western white people) and free for anyone to take and use, _when in fact_ those things still belong to the cultures which produce them, even if individual copyright law does not apply. White people generally do not have similar fears about aspects of their culture being appropriated by the entire world because in their view, their culture _owns_ the entire world.